Mathlib/Topology/PartitionOfUnity.lean

```diff
@@ -313,6 +313,9 @@ instance : FunLike (BumpCovering ι X s) ι C(X, ℝ) where
   coe := toFun
   coe_injective' f g h := by cases f; cases g; congr
 
+@[simp]
+lemma apply_coe{i : ι} : f i = f.toFun i := rfl
```

Comment:
This shouldn't be a `simp` lemma: `DFunLike.coe` is the simp normal form, not `toFun`.

Reply:
sorry, I removed `simp`

Reply:
This lemma should be stated in the other direction as `toFun_eq_coe`, and tagged `@[simp]`.

Reply:
ah I see! done